runtime/rasdump/trigger.c

```diff
@@ -1002,7 +1002,38 @@ triggerDumpAgents(struct J9JavaVM *vm, struct J9VMThread *self, UDATA eventFlags
 								OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
 								char dateStamp[64];
 								omrstr_ftime_ex(dateStamp, sizeof(dateStamp), "%Y/%m/%d %H:%M:%S", now, OMRSTR_FTIME_FLAG_LOCAL);
-								j9nls_printf(PORTLIB, J9NLS_INFO | J9NLS_STDERR | J9NLS_VITAL, J9NLS_DMP_PROCESSING_EVENT_TIME, mapDumpEvent(eventFlags), detailLength, detailData, dateStamp);
+
+								/*If there is more details about the event, print it.
+								 * During abort event detailData is empty string - skip the abort event */
+								if (0 != strcmp(detailData, "") && (NULL != eventData->exceptionRef) && (NULL != *eventData->exceptionRef)) {
+									j9object_t emessage = J9VMJAVALANGTHROWABLE_DETAILMESSAGE(self, *eventData->exceptionRef);
+									if (NULL != emessage) {
+										char stackBuffer[256];
+										char* buf = stackBuffer;
+										UDATA len = 0;
+										UDATA subLength = strlen(detailData) + strlen("\" \"");
+										char* subDetailedMessage = (char *)j9mem_allocate_memory(subLength+1, OMRMEM_CATEGORY_VM);
+										if (NULL != subDetailedMessage) {
+											strcpy(subDetailedMessage, detailData);
+											strcat(subDetailedMessage, "\" \"");
+											buf = self->javaVM->internalVMFunctions->copyStringToUTF8WithMemAlloc(self, emessage, J9_STR_NULL_TERMINATE_RESULT, subDetailedMessage, subLength, stackBuffer, 256, &len);
+											if (NULL != buf) {
+												j9nls_printf(PORTLIB, J9NLS_INFO | J9NLS_STDERR | J9NLS_VITAL, J9NLS_DMP_PROCESSING_EVENT_TIME, mapDumpEvent(eventFlags), strlen(buf), buf, dateStamp);
```

Comment:
This should use `len` (computed by `copyStringToUTF8WithMemAlloc()`) instead of `strlen(buf)`.
This line also too long, as are the other calls to `j9nls_printf()`.